packages/gridfs/package.js

```diff
@@ -15,12 +15,12 @@ Package.onUse(function(api) {
   api.versionsFrom('1.0');
 
   api.use(['cfs:base-package@0.0.27', 'cfs:storage-adapter@0.1.1']);
-  api.addFiles('gridfs.server.js', 'server');
-  api.addFiles('gridfs.client.js', 'client');
+  api.addFiles('packages/gridfs/gridfs.server.js', 'server');
+  api.addFiles('packages/gridfs/gridfs.client.js', 'client');
 });
 
 Package.onTest(function(api) {
   api.use(['cfs:gridfs', 'test-helpers', 'tinytest'], 'server');
-  api.addFiles('tests/server-tests.js', 'server');
-  api.addFiles('tests/client-tests.js', 'client');
+  api.addFiles('packages/gridfs/tests/server-tests.js', 'server');
+  api.addFiles('packages/gridfs/tests/client-tests.js', 'client');
```

Comment:
@raix see discussion in https://github.com/meteor/windows-preview/issues/45. It's because I published 0.0.28 with the windows preview release, which is apparently broken. It sounds like @stubailo is investigating, but maybe we should just publish (and publish-for-arch) a 0.0.29 release using the normal meteor release in the meantime.